src/Groups/GAPGroups.jl

```diff
@@ -36,7 +36,6 @@ export
     isperfect,
     ispgroup,
     issimple,
-    listperm,
```

Comment:
Actually I wrote that I'd rather *not* remove it but rather deprecate it... Anyway, whatever, we'll remove it at some point, so we can also do it now; but then really update *all* references to it; if you search the Oscar.jl repository for `listperm`, (by e.g. going in a terminal to its top directory, and then entering `git grep -n listperm`, you'll see that it also occurs in `docs/src/Groups/groups.md` and `examples/Invariants.jl` so these files also need to be adjusted.

Reply:
So, do I leave the definition of `listperm` and its export in `GAPGroups.jl`, but I remove all the references?